private/buf/cmd/buf/command/generate/generate_test.go

```diff
@@ -222,7 +222,6 @@ func testCompareGeneratedStubsArchive(
 		actualProtocPluginFlags...,
 	)
 	genFlags := []string{
-		"--input",
```

Comment:
Note that this flag might have been intentionally there to test that the deprecated flag code path continues to work.

Reply:
The test was not asserting anything around the message, but I can restore this is this was the intention.